static/docs/get-started/agenda.md

```diff
@@ -1,36 +1,35 @@
 # Agenda
 
-In the next few sections we will build a simple natural language processing
```

Comment:
Question for you @gagejustins, what criteria do you recommend to decide on abbreviations such as "we'll", "they're", "we've", etc? We haven't come up with a definite "policy" so this is currently a (minor) problem throughout our docs. I think that we should NOT (vs. shouldn't 😋 ) abbreviate when the sentence needs to emphasize the 2nd word in the abbreviation, as long as it's an idea directly related to the document's topic. What do you think?

> We also some times may use large caps as in "should NOT" which I think should be avoided in the future once this policy is decided.

Reply:
Great question! I think abbreviations for most pronouns tend to be more personal and inviting, which definitely vibes with DVC's brand. For emphasis I'd use italics, and agree that we shouldn't abbreviate in that case

Reply:
Agree! Let's use this practice @shcheklein? Except... not sure we want to start using _italics_ (as mentioned in one of my review comments below) – mainly because there's so many other docs we'd have to review and add them to in order to be consistent 😅. Oh, and we tend to use **bold** words more, although also not super consistently...

Reply:
Gotcha, will adjust to using bold! An important thing to add for docs (at least in my experience) is that we'll never be perfect, and users rarely expect you to be. In fact, DVC's docs are _remarkably_ good and y'all are a tiny company

Reply:
Deciding a word-emphasizing policy/ best practice and reviewing existing docs seems to me an issue of its own so I opened #686 for now.